baseapp/server.go

```diff
@@ -79,7 +79,22 @@ func NewServer(c HTTPConfig, params ...Param) (*Server, error) {
 	if base.server == nil {
 		base.server = &http.Server{
 			TLSConfig: &tls.Config{
-				MinVersion: tls.VersionTLS12,
+				MinVersion:               tls.VersionTLS12,
+				PreferServerCipherSuites: true,
```

Comment:
What's the reason for setting this to `true`? The Mozilla page linked below recommends allowing clients to chose their preference from the list supported by the server.

Reply:
fixed